packages/amplication-client/src/Resource/ResourceHome.tsx

```diff
@@ -46,26 +43,37 @@ const ResourceHome = ({ match, innerRoutes }: Props) => {
               pageTitle={currentResource?.name}
             >
               <div
-                className={classNames(
-                  `${CLASS_NAME}__header`,
-                  `theme-${
-                    currentResource && COLOR_TO_NAME[currentResource?.color]
-                  }`
-                )}
+                className={`${CLASS_NAME}__header`}
+                style={{
+                  backgroundColor:
+                    resourceThemeMap[currentResource?.resourceType].color,
+                }}
```

Comment:
Will this show a solid color for the background?

Reply:
The background looks the same as now